src/connectors/webauthn-fallback.ts

```diff
@@ -19,7 +19,10 @@ document.addEventListener('DOMContentLoaded', async () => {
 
     document.getElementById('msg').innerText = translate('webAuthnFallbackMsg');
     document.getElementById('remember-label').innerText = translate('rememberMe');
-    document.getElementById('webauthn-button').innerText = translate('webAuthnAuthenticate');
```

Comment:
Why remove the button text?

Reply:
It's set on line 24. I extracted the button to a variable.